sklearn/svm/_base.py

```diff
@@ -144,6 +144,13 @@ def fit(self, X, y, sample_weight=None):
             raise TypeError("Sparse precomputed kernels are not supported.")
         self._sparse = sparse and not callable(self.kernel)
 
+        if hasattr(self, 'decision_function_shape'):
+            if self.decision_function_shape not in ('ovr', 'ovo'):
+                raise ValueError(
+                    f"decision_function_shape must be either 'ovr' or 'ovo', "
+                    f"got {self.decision_function_shape}."
```

Comment:
silly question, when you do an multi line f-string, does the `f` need to be present at every line, even if that line doesn't "need" it?

Reply:
I think we only need `f` if we're using  brackets on that line. I don't know what the recommended way is. I feel like putting f everywhere is more consistent and everything is aligned, but no strong opinion